jmeter/jmeter-java/src/test/java/com/baeldung/jmeter/java/JMeterFromJavaUnitTest.java

```diff
@@ -0,0 +1,103 @@
+package com.baeldung.jmeter.java;
+
+import java.io.File;
+import java.net.URL;
+
+import org.apache.jmeter.control.LoopController;
+import org.apache.jmeter.engine.StandardJMeterEngine;
+import org.apache.jmeter.extractor.JSR223PostProcessor;
+import org.apache.jmeter.protocol.http.sampler.HTTPSampler;
+import org.apache.jmeter.save.SaveService;
+import org.apache.jmeter.testelement.TestPlan;
+import org.apache.jmeter.threads.SetupThreadGroup;
+import org.apache.jmeter.util.JMeterUtils;
+import org.apache.jorphan.collections.HashTree;
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Disabled;
+import org.junit.jupiter.api.Test;
+
+import com.github.tomakehurst.wiremock.WireMockServer;
+import com.github.tomakehurst.wiremock.client.WireMock;
+import com.github.tomakehurst.wiremock.core.WireMockConfiguration;
+
+class JMeterFromJavaUnitTest {
+
+    private WireMockServer wireMockServer = new WireMockServer(new WireMockConfiguration().port(8080));
+
+    @BeforeEach
+    void setUp() {
+        wireMockServer.start();
+        WireMock.configureFor("localhost", 8080);
+        WireMock.stubFor(WireMock.get("/baeldung")
+            .willReturn(WireMock.aResponse()
+                .withBody("Hello from baeldung server.")
+                .withStatus(200)));
+    }
+
+    @AfterEach
+    void tearDown() {
+        wireMockServer.stop();
+    }
+
+    @Test
+    @Disabled("enable after adding link to jmeter installation")
+    void runExistingJMeterTest() throws Exception {
+        StandardJMeterEngine standardJMeterEngine = new StandardJMeterEngine();
+
+        JMeterUtils.loadJMeterProperties("path-to-jmeter-installation/bin/jmeter.properties");
+        JMeterUtils.setJMeterHome("path-to-jmeter-installation/apache-jmeter-5.3");
+        JMeterUtils.initLocale();
+
+        SaveService.loadProperties();
+
+        URL resource = getClass().getClassLoader()
+            .getResource("test_plan.jmx");
```

Comment:
This may be one of those times where it's ok to use `src/test/resources` to construct a filename.